### ISSUE TITLE
Sorting Alphabetically

### ISSUE THREAD

**Issue by rubengarciam** [realm/realm-js#1149]
Sorting by items like names, etc are not returned in alphabetical order.

For example, if a collection contains Items with titles "item #1", item #2", item #10", "item #11", the output will be:
- item #1
- item #10
- item #11
- item #2

When instead it should be
- item #1
- item #2
- item #10
- item #11

Is the sorting not taking into consideration the length of the item?

**Comment by kristiandupont** [comment]
You are right, sorting doesn't do this currently. I agree that it would be nice -- we'll keep it in mind.

**Comment by mjm918** [comment]
Any update on this?
